src/main/java/com/mamu/todo_app/helper/EnumNames.java

```diff
@@ -0,0 +1,18 @@
+package com.mamu.todo_app.helper;
+
+import com.mamu.todo_app.types.StatusType;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Component
+public class EnumNames {
+    public List<String> getEnumNames(){
```

Comment:
Why do you need a new class for this? since you are only using StatusTye probably you can move it into StatusType

Reply:
I made changes in the code so that this class is no longer necessary